docs/kind.md

```diff
@@ -82,32 +82,40 @@ configuration options when deploying. Use `./kind.sh -h` to see the latest optio
 
 ```
 [root@ovnkubernetes contrib]# ./kind.sh --help
-usage: kind.sh [[[-cf |--config-file <file>] [-kt|keep-taint] [-ha|--ha-enabled]
+usage: kind.sh [[[-cf |--config-file <file>] [-kt|--keep-taint] [-ha|--ha-enabled]
                  [-ho |--hybrid-enabled] [-ii|--install-ingress] [-n4|--no-ipv4]
                  [-i6 |--ipv6] [-wk|--num-workers <num>] [-ds|--disable-snat-multiple-gws]
-                 [-dp |--disable-pkt-mtu-check] [-df|--disable-forwarding]
+                 [-dp |--disable-pkt-mtu-check]
+                 [-df |--disable-forwarding]
                  [-nf |--netflow-targets <targets>] [sf|--sflow-targets <targets>]
-                 [-if |--ipfix-targets <targets>] [-ifs|--ipfix-sampling <num>]
+                 [-if |--ipfix-targets <targets>]  [-ifs|--ipfix-sampling <num>]
                  [-ifm|--ipfix-cache-max-flows <num>] [-ifa|--ipfix-cache-active-timeout <num>]
                  [-sw |--allow-system-writes] [-gm|--gateway-mode <mode>]
                  [-nl |--node-loglevel <num>] [-ml|--master-loglevel <num>]
                  [-dbl|--dbchecker-loglevel <num>] [-ndl|--ovn-loglevel-northd <loglevel>]
                  [-nbl|--ovn-loglevel-nb <loglevel>] [-sbl|--ovn-loglevel-sb <loglevel>]
                  [-cl |--ovn-loglevel-controller <loglevel>]
                  [-ep |--experimental-provider <name>] |
-                 [-eb |--egress-gw-separate-bridge]
+                 [-eb |--egress-gw-separate-bridge] |
+                 [-lr |--local-kind-registry |
+                 [-dd |--dns-domain |
+                 [-ric | --run-in-container |
+                 [-cn | --cluster-name |
+                 [-ehp|--egress-ip-healthcheck-port <num>]
+                 [-is | --ipsec]
+                 [-cm | --compact-mode]
+                 [--isolated]
                  [-h]]
 
 -cf  | --config-file                Name of the KIND J2 configuration file.
                                     DEFAULT: ./kind.yaml.j2
 -kt  | --keep-taint                 Do not remove taint components.
                                     DEFAULT: Remove taint components.
 -ha  | --ha-enabled                 Enable high availability. DEFAULT: HA Disabled.
--scm | --separate-cluster-manager   Separate cluster manager from ovnkube-master and run as a separate container within ovnkube-master deployment.
```

Comment:
why was this removed?

Reply:
I just copied the output of `kind.sh --help`. This parameter was not added to the help out.

Reply:
I fixed it.